config/prysm/beacon.yaml

```diff
@@ -17,6 +17,8 @@ p2p-host-dns: ""
 rpc-host: 0.0.0.0
 monitoring-host: 0.0.0.0
 
+grpc-gateway-corsdomain: http://beacon-chain:7500
```

Comment:
I guess this doesn't belong in this pull request, right? It's for prysm

Reply:
yes, it is for prysm